src/aap_eda/settings/default.py

```diff
@@ -445,3 +446,13 @@ def _get_secret_key() -> str:
     os.path.dirname(dynamic_config.__file__), "dynamic_settings.py"
 )
 include(dab_settings)
+
+# ---------------------------------------------------------
+# DJANGO ANSIBLE BASE JWT SETTINGS
+# ---------------------------------------------------------
+ANSIBLE_BASE_JWT_VALIDATE_CERT = settings.get(
```

Comment:
@Dostonbek1 Is the JWT support in DAB going to support getting refresh tokens. EDA is using Simple JWT for authenticating web sockets connection and getting refresh tokens.

Reply:
@Dostonbek1 same question. Can you explain why we need to set these two settings?